### PR TITLE
Issue #SB-30052 fix: Refactoring - Proxy server code for Editors & Players

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: config/constants.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    "URLS": {
```
Comment (vinukumar-vs) [inline config/constants.js:2]: Rename URLS to API

File: proxyUtils.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const envVariables = require('./config/environment');
+
+const decoratePrivateRequestHeaders = function () {
+  return function (proxyReqOpts, srcReq) {
+    proxyReqOpts.headers["Content-Type"] = "application/json";
```
Comment (vinukumar-vs) [inline proxyUtils.js:5]: Why dont you call decoratePublicRequestHeaders & then append specific to headrs like userToken & channelId

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: proxyUtils.js (hunk 2 above)
Comment (vaibhavbhuva) [reply proxyUtils.js:5]: @vinukumar-vs resolved.

File: config/constants.js (hunk 1 above)
Comment (vaibhavbhuva) [reply config/constants.js]: Done

**Pull request merged**